### DISCUSSION TITLE
Sum two columns fields

### DISCUSSION THREAD

**Discussion by EduardoZeroum** [Budibase/budibase#5693] [Help] [answered]
Hello , im new at budibase and i need help with the {{add}} function because i want to multiply the quantity of the product and the value of them and add 24% of IVA. How can i manage to use the table variables like the {{Value}} one. 

Thank you in advance,
Eduardo Silveira.

**Comment by melohagan** [comment, marked as answer]
Hi @EduardoZeroum 

Might be easiest to use a JavaScript binding: https://docs.budibase.com/docs/javascript
On a  formula column: https://docs.budibase.com/docs/formula

**Reply by EduardoZeroum** [reply]
Thank you very much <3
